src/components/contextual/pages/pool/PoolBalancesCard/components/AssetRow.vue

```diff
@@ -33,14 +39,29 @@ const { explorerLinks } = useWeb3();
  */
 const token = computed(() => getToken(props.address));
 
-const balance = computed(() =>
-  formatUnits(props.balance, token.value.decimals)
-);
+const balance = computed(() => {
+  const formattedBalance = formatUnits(props.balance, token.value.decimals);
+  return props.share != null
+    ? bnum(formattedBalance)
+        .times(props.share)
+        .toString()
+    : formattedBalance;
+});
 
 const balanceLabel = computed(() => fNum(balance.value, 'token'));
 
 const fiatLabel = computed(() => {
-  const fiatValue = toFiat(balance.value, props.address);
+  let fiatValue = toFiat(
+    balance.value,
+    props.mainTokenAddress ?? props.address
+  );
+
+  if (props.priceRate != null) {
+    fiatValue = bnum(fiatValue)
+      .times(props.priceRate)
+      .toString();
+  }
```

Comment:
I guess it might not make a difference, but I think the `priceRate` should be applied to the balance and not the fiat value.

Reply:
Agree that for increased consistency `priceRate` should be applied to the balance too. For example, here we're saying the pool holds 10,133 aUSDT. 
![image](https://user-images.githubusercontent.com/34865315/145988437-92d2cd5c-8d31-4a41-873b-502c952c4327.png)

In reality it holds 10,133 **waUSDT**, so we need to apply the rate to get the aUSDT equivalent